.github/workflows/WorkitemValidation.yaml

```diff
@@ -49,6 +49,13 @@ jobs:
         run: |
           build/scripts/PullRequestValidation/ValidateInternalWorkItemForPullRequest.ps1 -PullRequestNumber ${{ github.event.pull_request.number }} -Repository ${{ github.repository }}
 
+      - name: Add Linked label to PR
+        if: github.event.pull_request.head.repo.full_name != github.repository
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          gh api /repos/${{ github.repository }}/issues/${{ github.event.pull_request.number }}/labels  -f "labels[]=Linked" -H "Accept: application/vnd.github.v3+json" -H "X-GitHub-Api-Version: 2022-11-28"
```

Comment:
Wouldn't that wipe all existing labels?

Reply:
No it's additive. It's the same thing we do with the "From Fork" label 
![image](https://github.com/microsoft/BCApps/assets/117829001/a375e083-858a-49b5-b5bb-bcaa47fc3fd4)